doc/classes/TabBar.xml

```diff
@@ -311,6 +311,9 @@
 		<theme_item name="font_disabled_color" data_type="color" type="Color" default="Color(0.875, 0.875, 0.875, 0.5)">
 			Font color of disabled tabs.
 		</theme_item>
+		<theme_item name="font_hovered_color" data_type="color" type="Color" default="Color(0.95, 0.95, 0.95, 1)">
+			Font color of the currently hovered tab.
```

Comment:
The style only applies to inactive hovered tabs. Hovering current tab doesn't use it. I think the description should clarify that.
Same below.

Reply:
Thank you for mentioning that point. I can make a change for the "active and hovered" tab state to have an option.

Reply:
I generally don't see a "active and hovered" tab state in other apps, so I think the current behavior from this PR is OK as well.